docs/gatsby-config.js

```diff
@@ -80,6 +80,8 @@ const themeOptions = {
               'analyze/index',
               'analyze/outcomes',
               'analyze/ceph-status',
+              'analyze/cluster-version',
+              "analyze/certificates",
```

Comment:
cluster-version is duplicated here.

Can we keep the quotes consistent?

Reply:
the quotes has been fixed, and duplicated has been removed